gematria/datasets/exegesis_benchmark.cc

```diff
@@ -259,11 +255,30 @@ Expected<double> benchmarkBasicBlock(const BenchmarkCode &BenchCode,
   return Result.Measurements[0].PerSnippetValue;
 }
 
+ExitOnError ExitOnErr("exegesis-benchmark error: ");
+
+// TODO(boomanaiden154): The function below and the following template are taken
+// from the llvm-exegesis code. These should be made generic and moved into
+// Error.h in upstream LLVM most likely. Check Err. If it's in a failure state
+// log the file error(s) and exit.
+static void ExitOnFileError(const Twine &FileName, Error Err) {
+  if (Err) {
+    ExitOnErr(createFileError(FileName, std::move(Err)));
+  }
+}
+
+// Check E. If it's in a success state then return the contained value.
+// If it's in a failure state log the file error(s) and exit.
+template <typename T>
+T ExitOnFileError(const Twine &FileName, Expected<T> &&E) {
```

Comment:
I guess these should be called `exitOnFileError()` to conform to the LLVM style function naming used in the rest of the file.